target/linux/ipq40xx/files-4.19/arch/arm/boot/dts/qcom-ipq4019-eap2200.dts

```diff
@@ -0,0 +1,282 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qcom-ipq4019.dtsi"
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	model = "EnGenius EAP2200";
+	compatible = "engenius,eap2200";
+
+	keys {
+			compatible = "gpio-keys";
+
+			wps {
+				label = "wps";
+				gpios = <&tlmm 18 GPIO_ACTIVE_LOW>;
+				linux,code = <KEY_WPS_BUTTON>;
+			};
+	};
+
+	aliases {
+		led-boot = &power;
+		led-failsafe = &power;
+		led-running = &power;
+		led-upgrade = &power;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		power: orange {
```

Comment:
Since these lables are being linked in the alias node, the latest craze is to add the "led_" prefix, so this could now look like

     led_power: power {
         ...
     }

Reply:
Thanks for comment. Modified.